app/assets/javascripts/comments.js

```diff
@@ -0,0 +1,29 @@
+$(document).on('turbolinks:load',function(){
+  load_message();
+  click_button_reply();
+  show_reply();
+})
+
+function load_message(){
+  $("#comments-link").click(function(){
+    $(".form-comments").fadeToggle();
+    $(".content-Post").focus();
+  });
+}
+
+function click_button_reply(){
+  $(".button-reply").click(function(){
```

Comment:
fix convetion đi em,
```suggestion
  $(".button-reply").click(function() {
```